MediaCV.css

```diff
@@ -1,100 +0,0 @@
-/* media queries */
-
-/********BREAKPOINTS********
-Moyenne résolution = 1024px
-Petite résolution = 767px
-****************************/
-
-
-@media screen and (max-width: 1024px)
-/* réduction de la taille des police, des icones et de la photo */
-{
-
-    body
-    {
-        width:auto;
-        font-size: 1.4rem
-    }
-
-    h1
-    {
-        font-size: 4rem;
-    }
-
-    #apropos p
-    {
-        text-align: left;
-        margin-left: 2rem;
-        margin-right: 2rem;
-    }
-
-    #loisirs i
-    {
-        border: 1.6px solid #212121;
-        border-radius: 50%;
-        color: #212121;
-        padding: 1.2rem;
-        width: 3rem;
-        height: 3rem;
-    }
-
-    img
-    {
-        width:70%;
-    }
-
-}
-
-@media screen and  (max-width: 767px)
-/* Affichage en colonne, réduction de la taille des police et des icones,
```

Comment:
Je te conseille de mettre ton commentaire au dessus de `@media screen and  (max-width: 767px)` pour éviter l'accolade toute seule à la ligne 51